### ISSUE TITLE
Package outline inconsistency in TD1204 Datasheet rev1.3.pdf

### ISSUE THREAD

**Issue by turdusmerula** [Telecom-Design/Documentation_TD_RF_Module#2]
The Figure 7 is wrong about pinning on each side, some pin groups were inverted.

Sebastien.

**Comment by mstempin** [comment]
Thank you for your feedback, but there is no figure 7 in TD1204 datasheet rev1.3 (08/14). However, figure 1 page 18 has its upper-right corner pins numbered wrong.

If this is not he inconsistency you are referring to, can you please provide a more detailed description?

**Comment by turdusmerula** [comment]
Sorry, I was talking about chapter 7 page 26. 
The group of 12 and 8 pins at bottom side of the bottom view should be at the up side.

**Comment by mstempin** [comment]
Yes, agreed, please use the figure 3 "PCB Land Pattern" instead of figure 2 "41-Pin Land Grid Array (LGA)" for correct footprint, until we correct the wrong drawing asap.

**Comment by mstempin** [comment]
Actually, only the group of 8 pins at the bottom right must be swapped with the group of 7 pins at the top right on the bottom view

**Comment by turdusmerula** [comment]
Is there an impact on measurements?

**Comment by athesi** [comment]
Dear, have you cad file to directly use the good footprint in Eagle ? Thanks

**Comment by smarthomeagentur** [comment]
eagle lib would be interessting for me too!
